docs/data/registry.json

```diff
@@ -35,6 +35,12 @@
       ]
     }
   },
+  "versions-dedicated-gen-3": {
```

Comment:
Looks like this is now a top-level key, when it should be within a service/runtime

Reply:
In trying to replicate this, placing `versions-dedicated-gen-3` back within `chrome-headless` fixed the local build. Registry parser was trying to find a `versions-dedicated-gen-3.docs.relationship_name`, which doesn't exist.

Reply:
Top of this file should be changed to:

```
{
  "chrome-headless": {
    "description": "",
    "disk": false,
    "docs": {
      "relationship_name": "chromeheadlessbrowser",
      "service_name": "headlessbrowser",
      "url": "/add-services/headless-chrome.html"
    },
    "endpoint": "http",
    "min_disk_size": null,
    "name": "Headless Chrome",
    "repo_name": "chrome-headless",
    "runtime": false,
    "type": "chrome-headless",
    "versions": {
      "deprecated": [],
      "supported": [
        "95",
        "91",
        "86",
        "84",
        "83",
        "81",
        "80",
        "73"
      ],
      "legacy": [
        "86",
        "84",
        "83",
        "81",
        "80",
        "73"
      ]
    },
    "versions-dedicated-gen-3": {
      "deprecated": [],
      "supported": [
        "95"
      ]
    }
  },
```

Reply:
cc @AnouckColson @remilejeune2